par/operations.py

```diff
@@ -1,16 +1,68 @@
 """Git and tmux operations - simplified from git.py and tmux.py"""
 
 import os
+import re
 import subprocess
 from pathlib import Path
 from typing import List, Optional
 
 import typer
 
 from .checkout import CheckoutStrategy
+from .constants import Config, Colors, Messages
 from .utils import get_git_repo_root, run_cmd
 
 
+# Input validation utilities
+def _validate_session_name(session_name: str) -> None:
+    """Validate tmux session name to prevent injection."""
```

Comment:
What's the injection case? Who would be injecting? Isn't it created by the user?